stories/covid-vax/README.md

```diff
@@ -0,0 +1,23 @@
+# covid-vax
+
+[![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gh/alan-turing-institute/TuringDataStories/covid-vax-forecast?filepath=stories%2Fcovid-vax%2Fcovid_vax.ipynb)
+
+Estimating when the UK will finish vaccinating the population. Use the link above
+to run the notebook in Binder, or to run it locally:
+
+- `cd` to the `covid-vax` directory.
+
+- Create the `conda` virtual environment:
+  ```
+  conda env create
+  ```
+
+- Activate the environment:
+  ```
+  conda activate covid-vax
+  ```
+
```

Comment:
How do you connect your conda environment with the jupyter notebook kernel? For me it doesn't happen by default (the default would still be Python 3, that doesn't even have pandas), so i had to use ipykernel in the way:

```
python -m ipykernel install --user --name=covid-vax
```

and then I select the kernel under that name. Is there another way? Let's make sure it's explicit.

Reply:
Weird, for me just running `jupyter-notebook` after `conda activate covid-vax` is enough to start jupyter with it defaulting to using the right python environment. Does `which jupyter-notebook` point to the `jupyter` installed in the virtual environment for you?

Is there a more reliable way to set it up? Very happy to change it if so (and probably should change some other things I have set up the same way if this doesn't always work 😀 !)

Reply:
Yeah, it points to the ```jupyter``` from the conda environment, . But it is strange, I have always had to do the ipykernel trick in all my notebooks and environments...

We can just leave it as it is, but add a small comment to the README saying to use the ipykernel trick if their notebooks are not directly associated to the conda environment? Maybe add this [link](https://gdcoder.com/how-to-create-and-add-a-conda-environment-as-jupyter-kernel/) as reference?

Reply:
P.S. for me running `jupyter kernelspec list` from the environment has the default `python3` kernel set to the virtual env:
```shell
jupyter kernelspec list
Available kernels:
  # ... other kernels, then ... 
  python3     /Users/jroberts/opt/anaconda3/envs/covid-vax/share/jupyter/kernels/python3
```

Reply:
Mine is a veeeery long list hehehe. Maybe is that if there is no other kernels defined the default goes to current conda environment?